overlord/configstate/configcore/hostname_test.go

```diff
@@ -51,10 +51,12 @@ func (s *hostnameSuite) SetUpTest(c *C) {
 }
 
 func (s *hostnameSuite) TestConfigureHostnameInvalid(c *C) {
+	filler := strings.Repeat("x", 60)
 	invalidHostnames := []string{
 		"-no-start-with-dash", "no-upper-A", "no-ä", "no/slash",
 		"ALL-CAPS-IS-NEVER-OKAY", "no-SHOUTING-allowed",
 		strings.Repeat("x", 64),
+		strings.Join([]string{filler, filler, filler, filler, filler}, "."),
```

Comment:
we should also check `"foo..bar"` since we shouldn't support an empty element between dots.